docs/release-notes/NuGet-6.8.md

```diff
@@ -0,0 +1,211 @@
+---
+title: NuGet 6.8 Release Notes
+description: Release notes for NuGet 6.8 including new features, bug fixes, and DCRs.
+author: <TODO: GitHubAlias>
+ms.author: <TODO: MicrosoftAlias>
+ms.date: 10/30/2023
+ms.topic: conceptual
+---
+
+# NuGet 6.8 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.8**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.8](https://visualstudio.microsoft.com/downloads/) | [<TODO: Full SDK Version>](https://dotnet.microsoft.com/download/dotnet-core/<TODO: SDKMajorMinorVersionOnly)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.8
+
+* NuGetAudit should warn when no vulnerability data available - [#12610](https://github.com/NuGet/Home/issues/12610)
+

```

Comment:
```suggestion
```

This feature was removed.  I just removed the issue from all milestones

Reply:
Thanks for doing that. 

We should probably be adding `Resolution:...` labels to issues that have basically been cut. 
Those get excluded from release notes.